scripts/lint.sh

```diff
@@ -148,9 +148,17 @@ function test_require_error_is_no_funcs_as_params {
 }
 
 function test_require_no_error_inline_func {
-  if grep -R -zo -P '\t+err :?= ((?!require|if).|\n)*require\.NoError\((t, )?err\)' "${DEFAULT_FILES[@]}"; then
+  # Flag only when a single variable whose name contains "err" or "Err"
```

Comment:
This modification was needed in this PR, because it was wrongly matching that functions with multiple return values should also be inlined which is not possible. Now the check skips those cases.